### PR TITLE
Separate pod memory usage by RSS/cache/swap

### PR DESCRIPTION
**Pull request by onitake**
This PR stacks the pod and namespace graphs, reporting RSS, cache and swap memory usage separately. The memory tables report RSS, cache and swap memory in addition to total memory usage per container/pod.

This addresses #152 .

### CODE REVIEW

**Comment by onitake** [comment]
The namespace memory graph looks more cluttered than the pod graph with separate memory usage, I wonder if reporting only RSS memory there makes more sense.

**Comment by metalmatze** [comment]
I would be ok with losing a bit of detail on the namespace memory graph to have better overview.
In the end, we want to have a "drill down" experience and I think it's still fine for that.

**Comment by onitake** [comment]
There, I think this looks better.

File: dashboards/resources.libsonnet [hunk 1]
```diff
@@ -213,8 +220,17 @@ local g = import 'grafana-builder/grafana.libsonnet';
         g.row('Memory Usage')
         .addPanel(
           g.panel('Memory Usage') +
-          g.queryPanel('sum(container_memory_usage_bytes{namespace="$namespace", pod_name="$pod", container_name!="POD", container_name!=""}) by (container_name)', '{{container_name}}') +
```
Comment (brancz) [inline dashboards/resources.libsonnet]: I think this should continue to use the full memory usage, for more advanced troubleshooting I feel users are going to either do their own queries or drill down where the pod dashboard does show the difference.

Comment (brancz) [reply dashboards/resources.libsonnet]: Sorry this was on the wrong line, I meant to comment on the namespace dashboard.

Comment (onitake) [reply dashboards/resources.libsonnet]: Ok, will change that.

**Comment by brancz** [comment]
lgtm :+1:

**Comment by onitake** [comment]
As you might have noticed, I took the liberty to change decbytes to bytes, so 2^10 units (MiB, GiB, etc.) are shown instead of 10^3 units (MB, GB).
If you prefer, I can move this to a separate PR.

**Comment by metalmatze** [comment]
I think it's fine within this PR. No worries. I'm giving it a last review as we speak :blush:

**Comment by metalmatze** [comment]
Looking really good. 

old:
old
new:
new

Thank you for contributing this!

**Pull request merged**